### DISCUSSION TITLE
Download falhou - O vídeo é baixado,  mas não mostra o conteúdo da aula, apenas o som.

### DISCUSSION THREAD

**Discussion by Magna3** [aclap-dev/video-downloadhelper#715] [Help and Q&A]
### Your question

Porque mudou a maneira de baixar as aulas? Antes era perfeito e agora apenas o som é retornado. Comprei a licença e não estou tendo sucesso com os downloads. A versão anterior era mais leve, rápida e eficaz. 
{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.udemy.com/course/cientista-de-dados/learn/lecture/27814446#overview",
  "error": {
    "details": "SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 0000029634aaab40] trun track id unknown, no tfhd was found\r\n[mov,mp4,m4a,3gp,3g2,mj2 @ 0000029634aaab40] error reading header\r\n[in#0 @ 0000029634aaa980] Error opening input: Invalid data found when processing input\r\nError opening input file https://dash-enc-c.udemycdn.com/out/v1/19212027ec8341c0910e322e984b01ce/91242a72c0fa403f8accd701a89c70a3/e8594a5d9a2a48f79d07fb6a75d109d4/61f2d120ed734c1eb770addea062c0a2/index_video_5_0_25.mp4?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9kYXNoLWVuYy1jLnVkZW15Y2RuLmNvbS9vdXQvdjEvMTkyMTIwMjdlYzgzNDFjMDkxMGUzMjJlOTg0YjAxY2UvOTEyNDJhNzJjMGZhNDAzZjhhY2NkNzAxYTg5YzcwYTMvZTg1OTRhNWQ5YTJhNDhmNzlkMDdmYjZhNzVkMTA5ZDQvNjFmMmQxMjBlZDczNGMxZWI3NzBhZGRlYTA2MmMwYTIvKiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTcxNjg0MjM1MH19fV19&Signature=AgM35as90U55CSMC5kBDpFk09xFMcSV7fnC2Gbut~9L97ui8LHlQdcp6PaHCOZO-kSEizyhzEGQ3hgL7XcHHAEha1RbZuPHcEpY8z~Su2M5SEIyemiLru6583GrwxQUohuRGFAN7mcNvdEckdhQV8ZwByy09IKhzSWyJeX2e0X1RJQjOiJB6Zf7FxQWpsctRYHRTzQ~Zkc4jax0hvjDHgVvkciJw54aq8n140b2fivZS7MOfYaLrnVIjzQAIycoeGnOPHJLzwjE0FHAexnZ-KBMJM4jLCZNLdrdaLQzAsrsSNF6VFAgyMVNhkZDnbwihRGo2kEhJQz0VmpQh~afWjg__&Key-Pair-Id=K3MG148K9RIRF4&CMCD=bl%3D29600%2Cbr%3D191%2Cd%3D6000%2Cdl%3D29600%2Cmtp%3D10500%2Cnor%3D%22index_video_5_0_26.mp4%22%2Cot%3Dv%2Crtp%3D200%2Csf%3Dd%2Csid%3D%22ca5becc8-505a-4310-ae2d-5fe903437ca2%22%2Cst%3Dv%2Ctb%3D473.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_5611028332170206",
    "error": "coapp_failure",
    "id": "coapp_failure_6f038a14-7a82-4677-9991-31f94c8b165e"
  }
}
Infelizmente, falhamos no download dessa mídia específica. Tentamos oferecer suporte ao maior número possível de sites, então ajudaria muito se você pudesse relatar esse erro!
![mídia baixada](https://github.com/aclap-dev/video-downloadhelper/assets/90223745/f5f284a5-df5c-4af5-b945-2cd402ac2568)
**### O arquivo quando é baixado, apresenta o som e não apresenta a imagem.**

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"key":"6921b06ddba34431","mismatch":true,"other_browser":"Firefox","this_browser":"Chrome"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 3

}
![situação atual]
(https://github.com/aclap-dev/video-downloadhelper/assets/90223745/defa825a-7be4-45c2-9760-77d28441c700)
![situação atual](https://github.com/aclap-dev/video-downloadhelper/assets/90223745/936627f0-cde2-42e2-8e5a-f2af864ae414)

**Comment by GrampaWildWilly** [comment]
Refer to #714.
